### PR TITLE
Implemented Delete function from within ShowFlashCardsDetails activit…

### PR DESCRIPTION
**Pull request by Cosmopal**
…y. had to do changes in the way a flash card is deleted. Flash cards are now deleted by calling appropriate function in the FlashCard class. Same is changed in the code

I am making this pull request to let you know how I will be implementing the edit feature. The edit feature will be a similar activity to ShowFalshCardsDetail activity, where there will be Edittextviews. If you are not happy with the changes in this, then I'll have to find out a different way

### CODE REVIEW

**Comment by Cosmopal** [comment]
@srv-twry Please reply with your review. If you merge it, I'll contniue on edit feature

**Comment by Cosmopal** [comment]
relates to #4 issue

File: app/src/main/java/com/example/srv_twry/studentcompanion/POJOs/FlashCard.java [hunk 1]
```diff
@@ -31,24 +47,108 @@ public String getAnswer() {
...
+            public void onClick(DialogInterface dialog, int which) {
+                dialog.cancel();
+            }
+        });
+
+        alertDialog.show();
+    }
+
+    public static FlashCard getFlashCardFromCursor(Cursor mCursor){
+        String tag = "getFlashcardfromcursor";
```
Comment (dr0pdb) [inline app/src/main/java/com/example/srv_twry/studentcompanion/POJOs/FlashCard.java:109]: I would suggest using standard Java convention of naming constants by all capitals.

**Review by dr0pdb — APPROVED** [review]
Other than one minute convention issue whick i would suggest you to take care from future PRs , everything looks good.
I think this method is okay and you can go ahead with the editing feature as well :smile:

**Pull request merged**